src/Concerns/HasUnits.php

```diff
@@ -6,11 +6,8 @@ trait HasUnits
 {
     /**
      * Define the total units count
-     *
-     * @param mixed $value
-     * @return $this
      */
-    public function setUnits($value)
+    public function setUnits(mixed $value): static
```

Comment:
(Not sure about this one)

```suggestion
    public function setUnits(float|int|string $value): static
```